tests/testthat/test-nonmem.R

```diff
@@ -211,7 +224,7 @@ test_that("tbs tests", {
       "     RX_IP1 = -1000000000",
       "  END IF",
       "  RX_P1 = RX_IP1",
-      "  W1=DSQRT((THETA(3))**2)",
```

Comment:
These are fine, if we are making readable code.

Should we mention which DADT() maps to what compartment?

I think the `$MODEL` does this, though.

Reply:
I'm still working to make the code more readable.  This is just the first step.

Generally, yes, I think that we should make the code more readable because it will make people more comfortable knowing that the fidelity in translation is good.